x-pack/plugins/security_solution/public/flyout/alerts/panels/right/content.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlyoutBody } from '@elastic/eui';
+import { css } from '@emotion/react';
+import type { VFC } from 'react';
+import React, { useMemo } from 'react';
+import type { RightPanelPaths } from '.';
+import { tabs } from './tabs';
+
+export interface PanelContentProps {
+  /**
+   * Id of the tab selected in the parent component to display its content
+   */
+  selectedTabId: RightPanelPaths;
+}
+
+/**
+ * Alert details expandable flyout right section, that will display the content
+ * of the overview, table and json tabs.
+ */
+export const PanelContent: VFC<PanelContentProps> = ({ selectedTabId }) => {
+  const selectedTabContent = useMemo(() => {
+    return tabs.find((tab) => tab.id === selectedTabId)?.content;
+  }, [selectedTabId]);
+
+  return (
+    <EuiFlyoutBody
+      css={css`
+        height: calc(100vh - 262px);
```

Comment:
I think I did the same thing with the explicit `262`, but I think we should see if we can get the topbar heights from some shared constant if possible in case that changes

Reply:
you're right again, I stole this from you without really thinking about why we're using it. I feel like this shouldn't be part of the PR, and I'll add it back in a cleaner way once I implement the content of the tabs!